### PR TITLE
Update README.md: Add Forewords

### PR DESCRIPTION
**Pull request by benjaoming**
Inspired by [another Awesome list](https://github.com/mojoaxel/awesome-regression-testing), I'm moving the motivational talk into Forewords.

LMK if you have any ideas that can make the list look awesome'er :) Here is one of the main lists of awesome lists where I intend to list this list: https://project-awesome.org/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,24 +19,28 @@ A list of example projects, useful to learn from and for bootstrapping new docum
 Read the Docs is a fully open-source platform that builds and publishes documentation. Read more
 on [readthedocs.org](https://readthedocs.org/) and [readthedocs.com](https://readthedocs.com/) (Read the Docs for Business).
 
-Documentation projects have traditionally been part of a software project. However, many new documentation projects have emerged in science and academia. Therefore, we are compiling a list of inspirational uses of documentation technology, especially outside of the traditional field of software documentation. It's a work in progress, please feel invited to [contribute](#contributing)!
+## Foreword
 
-<!-- TOC -->
+Documentation projects have traditionally been part of a software project. However, many new documentation projects have emerged in science and academia. To capture the latest development and trends, we are compiling a list of inspirational uses of documentation technology, especially outside of the traditional field of software documentation.
+
+<!-- we never enable the double-link rule again, it's annoying -->
```
Comment (ericholscher) [inline README.md]: This comment doesn't make sense without context?

**Review by ericholscher — APPROVED** [review]
I'll be curious if any of these external channels bring traffic. I also think it's a bit weird that it's in a repo called `.github`, instead of `awesome-readthedocs`.

File: README.md (hunk 1 above)
Comment (benjaoming) [reply README.md]: The comment below disables the linting rule, so this was just to clarify that we don't re-enable it, it remains off. Might actually just move it to the top.

**Comment by benjaoming** [comment]
> I also think it's a bit weird that it's in a repo called .github, instead of awesome-readthedocs.

agreed, the idea was to have a repo that became the frontpage of https://github.com/readthedocs-examples/

**Comment by ericholscher** [comment]
> > I also think it's a bit weird that it's in a repo called .github, instead of awesome-readthedocs.
> 
> agreed, the idea was to have a repo that became the frontpage of https://github.com/readthedocs-examples/

Yea, just not sure that it's going to get as much traffic there? Probably worth re-thinking since that page is really busy, and we probably want to highlight the repo itself, not the organization.

**Comment by benjaoming** [comment]
Another issue would be if the organization index page actually has any traffic analysis... a repo has statistics that we already know how to understand.

**Comment by benjaoming** [comment]
It seems like we agree to rename the repo before adding it to any "Awesome List" lists :+1:

**Comment by benjaoming** [comment]
The repo is renamed :+1:

**Comment by benjaoming** [comment]
I just gave this a re-read, and I think it would be nice to get this in before continuing towards a broader announcement.

**Pull request merged**